doc/book/getting-started/database-and-models.md

```diff
@@ -217,10 +217,10 @@ class Module implements ConfigProviderInterface
         return [
             'factories' => [
                 Model\AlbumTable::class =>  function($container) {
-                    $tableGateway = $container->get(Model\AlbumTableGateway::class);
+                    $tableGateway = $container->get('Model\AlbumTableGateway');
```

Comment:
I tried to replicate it, but need to start the tutorial from scratch since I didn't include the config files with the commits. The issue was that the file isn't there and it caused a weird error. Can't remember what it was exactly. When I have time I'll I'll replicate it and will create a new issue.

The tutorial state at that point: https://github.com/xtreamwayz/zf3-album-tutorial/tree/3ef54dbcad8bd84b2c0f7203f4d6436ac751e214